<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>621</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>497</width>
    <height>621</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>497</width>
    <height>621</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>24</red>
        <green>24</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>12</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Yandex.Music RPC</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>20</y>
      <width>421</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffcc00;&quot;&gt;Яндекс.Музыка &lt;/span&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Discord RPC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QFrame" name="top_container">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Яндекс.Музыка OAuth Токен</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight|Qt::AlignVCenter">
             <widget class="QPushButton" name="help_btn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources/res.qrc">
                <normaloff>:/icons/help_icon.svg</normaloff>:/icons/help_icon.svg</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="token_input">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius: 2px;
width: 100%;
padding: 5px 10px;</string>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Roboto</family>
                <pointsize>12</pointsize>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Задержка обновления треков</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignRight|Qt::AlignVCenter">
             <widget class="QLabel" name="delay_label">
              <property name="text">
               <string>5 сек.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QSlider" name="delay_slider">
            <property name="styleSheet">
             <string notr="true">QSlider{
	height: 30px;
}
QSlider::groove {
    height: 20px;
    margin: 0px;
	border-radius: 10px;
    background-color: white;
}
QSlider::handle{
	border: 5px solid #FFCC00;
	background: white;
	height: 20px;
    width: 20px;
    border-radius: 15px;
	margin: -5px 0;
}</string>
            </property>
            <property name="minimum">
             <number>5</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="sliderPosition">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Показывать кнопку с ссылкой на текущий трек?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background: #FFCC00;
	text-transform: uppercase;
	font-size: 16px;
	font-family: &quot;Roboto&quot;, sans-serif;
	font-weight: normal;
	padding: 10px 15px;
	border-radius: 2px;
}
QPushButton::hover{
	background-color: rgb(225, 180, 0);
}</string>
          </property>
          <property name="text">
           <string>Обновить / Сохранить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QFrame" name="bottom_container">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">background: #131313;
font-size: 14px;
font-family: 'Roboto';
font-weight: normal;
padding: 10px 5px;
border-radius: 2px;
font-size: 16px;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">color: white;
margin: 0;
padding: 0;</string>
             </property>
             <property name="text">
              <string>Текущий статус</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight|Qt::AlignVCenter">
            <widget class="QLabel" name="status_label">
             <property name="font">
              <font>
               <family>Roboto</family>
               <pointsize>-1</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: #FFCC00;
font-weight: bold;
margin: 0;
padding: 0;</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gh_btn">
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background: #131313;
	font-size: 16px;
	font-family: 'Roboto';
	font-weight: normal;
	padding: 10px 5px;
	border-radius: 2px;
	color: white;
}
QPushButton::hover{
	background: rgb(255, 204, 0);
}</string>
          </property>
          <property name="text">
           <string>  Исходный код</string>
          </property>
          <property name="icon">
           <iconset resource="resources/res.qrc">
            <normaloff>:/icons/github_icon.svg</normaloff>:/icons/github_icon.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Aleksandr &lt;waitstop&gt; Kazakov</string>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>gh_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_gh_link()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>help_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delay_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>slider_changed_value(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>handle_save_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>handle_checkbox(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_gh_link()</slot>
  <slot>open_help()</slot>
  <slot>slider_changed_value(int)</slot>
  <slot>handle_save_btn()</slot>
  <slot>handle_checkbox(int)</slot>
 </slots>
</ui>
